jobsdb/integration_test.go

```diff
@@ -148,7 +149,7 @@ func TestJobsDB(t *testing.T) {
 		CustomVal: true,
 	}
 
-	jobDB.Setup(jobsdb.ReadWrite, false, "batch_rt", dbRetention, migrationMode, true, queryFilters)
+	jobDB.Setup(jobsdb.ReadWrite, false, "batch_rt", dbRetention, migrationMode, true, queryFilters, []prebackup.Handler{})
```

Comment:
I wonder if we could just pass `nil` instead of an empty slice here. It shouldn't break your code.

Reply:
Because it is easier to read, an empty slice of `prebackup.Handler` is more explicit than a nil which can be anything. Performance here is not an issue.

Reply:
What do you mean it can be anything? You can't pass anything there, it can either be `nil` or a slice of that type or it won't compile, right? In any case, minor optimization but I don't see the reason to allocate. Plus it is a test so completely up to you man :+1:

Reply:
if you were to only read this line, ignoring the actual method signature, you can now understand that the caller passes an empty slice of `prebackup.Handler`. If this changes to `nil`, you won't understand much until you go and look at the method's actual signature.